faq/building.inc

```diff
@@ -1744,3 +1744,19 @@ href=\"$topdir/community/help/\">Getting Help</a> page -- it details
 how to send a request to the Open MPI mailing lists.";
 
 /////////////////////////////////////////////////////////////////////////
+
+$q[] = "How do I statically link to the libraries of Intel compiler suite?";
+
+$anchor[] = "statically-link-with-intel-libraries";
+
+$a[] = "When compiled with Intel compiler suite, by default Openmpi binaries are
```

Comment:
"Open MPI", not "Openmpi".

I'd change the first sentence to:

> The Intel compiler suite, by default, dynamically links its runtime libraries against the Open MPI binaries and libraries.  This can cause problems if the Intel compiler libraries are installed in non-standard locations.  For example, you might get errors like:
> 
> ...put verbatim error messages here...
>
> To avoid such problems, you can pass flags to Open MPI's configure script that instruct the Intel compiler suite to statically link its runtime libraries with Open MPI:
>
> ...geshi bash configure example...